app/src/main/java/com/teamagam/gimelgimel/app/map/esri/EsriGGMapView.java

```diff
@@ -59,6 +62,20 @@ public void setGGMapGestureListener(OnMapGestureListener onMapGestureListener) {
     }
 
     @Override
+    public void saveState() {
+        SharedPreferences prefs = getDefaultSharedPreferences();
```

Comment:
UserPreferenceRepository is used inside domain (and possibly data, even-though it doesn't have to).
SharedPreferences can be used app (it's even better IMO).